### PR TITLE
fix(empty-array-and-address): fix empty array issue and change address

### PR DESCRIPTION
**Pull request by varun257**
### Description of the Change

Update new address
Make sure no results is shown in datagrid

### Benefits

Show no result in table

### Possible Drawbacks

None

### Applicable Issues

resolves MBE-107, MBE-172

### CODE REVIEW

**Comment by varun257** [comment]
Cant get a build of telecom. So QA wont pass. Will reopen once i get a build.

File: client/app/telecom/telephony/line/phone/order/translations/Messages_fr_FR.xml [hunk 1]
```diff
@@ -25,7 +25,7 @@
    <translation id="telephony_line_phone_order_process_exchange2" qtlid="459040">Vous payez la caution du nouveau matériel lors de la commande. Celle de l'ancien matériel vous est restituée sous 10 jours après réception de l'équipement par nos services.</translation>
    <translation id="telephony_line_phone_order_return_phone" qtlid="459053">Renvoyer mon équipement actuel ({{ phone }})</translation>
    <translation id="telephony_line_phone_order_return_address_title" qtlid="459066">Adresse de retour :</translation>
-   <translation id="telephony_line_phone_order_return_address_content" qtlid="459079">OVH SAS<br />Service Après Vente<br />13 rue Kellermann<br />59100 Roubaix</translation>
+   <translation id="telephony_line_phone_order_return_address_content">OVH SAS<br />Service Après Vente<br />155 avenue Georges Hannart<br />59170 Croix</translation>
```
Comment (marie-j) [inline client/app/telecom/telephony/line/phone/order/translations/Messages_fr_FR.xml]: Could the `<br/>` be removed and the translation split into chunks ?

Comment (varun257) [reply client/app/telecom/telephony/line/phone/order/translations/Messages_fr_FR.xml]: Please check if this is okay ? If `<br>` has to be avoided altogether, I can add a class and `<p>` tag

File: client/app/telecom/telephony/line/phone/order/telecom-telephony-line-phone-order.html [hunk 2]
```diff
@@ -493,7 +493,10 @@ <h3 data-translate="telephony_line_phone_order_summary"></h3>
                 </p>
                 <p data-ng-if="PhoneOrderCtrl.phone">
                     <strong data-translate="telephony_line_phone_order_return_address_title"></strong><br>
-                    <span data-translate="telephony_line_phone_order_return_address_content"></span>
+                    <span data-translate="telephony_line_phone_order_return_address_content_name"></span><br>
```
Comment (JeremyDec) [inline client/app/telecom/telephony/line/phone/order/telecom-telephony-line-phone-order.html]: To remove br tag, you can :
```suggestion
                    <span class="d-block" data-translate="telephony_line_phone_order_return_address_content_name"></span>
```

Comment (JeremyDec) [inline client/app/telecom/telephony/line/phone/order/telecom-telephony-line-phone-order.html]: To remove br tag, you can, (and apply this to `strong` tag as well) :
```suggestion
                    <span class="d-block" data-translate="telephony_line_phone_order_return_address_content_name"></span>
```

Comment (marie-j) [reply client/app/telecom/telephony/line/phone/order/telecom-telephony-line-phone-order.html]: Or 
```suggestion
                    <p data-translate="telephony_line_phone_order_return_address_content_name"></p>
```

Comment (varun257) [reply client/app/telecom/telephony/line/phone/order/telecom-telephony-line-phone-order.html]: `<p>` tags have margin by default so going with `<span>`

File: client/app/telecom/telephony/line/management/terminate/telecom-telephony-line-management-terminate.html [hunk 3]
```diff
@@ -33,7 +33,10 @@
                             <i class="ovh-font ovh-font-home ovh-font-1x8 pt-3" aria-hidden="true"></i>
                             <div class="pl-5">
                                 <h4 data-translate="telephony_group_line_terminate_back_address"></h4>
-                                <span translate="telephony_group_line_terminate_address"></span>
+                                <span class="d-block" data-translate="telephony_group_line_terminate_address_name">OVH SAS</span>
```
Comment (antleblanc) [inline client/app/telecom/telephony/line/management/terminate/telecom-telephony-line-management-terminate.html]: For that purpose you can use the `<address>` element I guess.

Comment (antleblanc) [reply client/app/telecom/telephony/line/management/terminate/telecom-telephony-line-management-terminate.html]: Please pay attention that you have to refer to the translation key only.

File: client/app/telecom/telephony/line/phone/order/telecom-telephony-line-phone-order.html [hunk 4]
```diff
@@ -492,8 +492,11 @@ <h3 data-translate="telephony_line_phone_order_summary"></h3>
                     </span>
                 </p>
                 <p data-ng-if="PhoneOrderCtrl.phone">
-                    <strong data-translate="telephony_line_phone_order_return_address_title"></strong><br>
-                    <span data-translate="telephony_line_phone_order_return_address_content"></span>
+                    <strong class="d-block" data-translate="telephony_line_phone_order_return_address_title"></strong>
```
Comment (antleblanc) [inline client/app/telecom/telephony/line/phone/order/telecom-telephony-line-phone-order.html]: For that purpose you can use the `<address>` element I guess.

File: client/app/telecom/telephony/line/management/terminate/telecom-telephony-line-management-terminate.html (hunk 3 above)
Comment (varun257) [reply client/app/telecom/telephony/line/management/terminate/telecom-telephony-line-management-terminate.html]: `<address>` element will again need `<br>` for line breaks

File: client/app/telecom/telephony/line/management/terminate/telecom-telephony-line-management-terminate.html [hunk 5]
```diff
@@ -33,7 +33,10 @@
                             <i class="ovh-font ovh-font-home ovh-font-1x8 pt-3" aria-hidden="true"></i>
                             <div class="pl-5">
                                 <h4 data-translate="telephony_group_line_terminate_back_address"></h4>
-                                <span translate="telephony_group_line_terminate_address"></span>
+                                <span class="d-block" data-translate="telephony_group_line_terminate_address_name"></span>
```
[the remark this answers is not in the document]
Comment (varun257) [reply client/app/telecom/telephony/line/management/terminate/telecom-telephony-line-management-terminate.html]: okay sure....have done this

**Pull request merged**